contracts/smart-account/paymasters/verifying/singleton/VerifyingSingletonPaymaster.sol

```diff
@@ -190,9 +194,19 @@ contract VerifyingSingletonPaymaster is
         UserOperation calldata userOp,
         bytes32 /*userOpHash*/,
         uint256 requiredPreFund
-    ) internal override returns (bytes memory context, uint256 validationData) {
+    )
+        internal
+        view
+        override
+        returns (bytes memory context, uint256 validationData)
```

Comment:
same here, context and validationData are two variables that are never used.

You should consider removing unused 'context' and 'validationData' variables for improved clarity or using it to return values to enhance code readability